docs/operator-manual/declarative-setup.md

```diff
@@ -84,6 +84,9 @@ kind: AppProject
 metadata:
   name: my-project
   namespace: argocd
+  # Finalizer that ensures that project is not deleted until it is referenced by any application
```

Comment:
Same here: until it is not referenced by any application

Reply:
done